OngProject/Core/Business/CategoryBusiness.cs

```diff
@@ -54,21 +54,10 @@ public async Task<CategoryFullDTO> Add(CategoryPostDTO newDTO)
             }
         }
 
-        public async Task<bool> Delete(int id)
+        public async Task Delete(Category entity)
```

Comment:
porque envias una entidad que es de la capa de datos?

Reply:
A DTO is an object that defines how the data will be sent over the network. En nuestra app no esta divida en micro servicios. los DTOS serian para evitar que toda la info llegue al cliente.